apps/member-profile/app/routes/_profile.profile.integrations.tsx

```diff
@@ -0,0 +1,133 @@
+import { json, type LoaderFunctionArgs } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+import { ExternalLink } from 'react-feather';
+
+import { Pill, Text } from '@oyster/ui';
+import { iife } from '@oyster/utils';
+
+import {
+  ProfileHeader,
+  ProfileSection,
+  ProfileTitle,
+} from '../shared/components/profile';
+import { ENV } from '../shared/constants.server';
+import { getMember } from '../shared/queries';
+import { ensureUserAuthenticated, user } from '../shared/session.server';
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const session = await ensureUserAuthenticated(request);
+
+  const id = user(session);
+
+  const member = await getMember(id)
+    .select(['githubId'])
+    .executeTakeFirstOrThrow();
+
+  const githubOauthUri = iife(() => {
+    const url = new URL('https://github.com/login/oauth/authorize');
+
+    url.searchParams.set('client_id', ENV.GITHUB_OAUTH_CLIENT_ID || '');
+
+    return url.toString();
+  });
+
+  return json({
+    isGithubConnected: !!member.githubId,
+    githubOauthUri,
+  });
+}
+
+export default function IntegrationsPage() {
+  const { githubOauthUri, isGithubConnected } = useLoaderData<typeof loader>();
+
+  return (
+    <ProfileSection>
+      <ProfileHeader>
+        <ProfileTitle>Integrations</ProfileTitle>
+      </ProfileHeader>
+
+      <ul className="grid grid-cols-1 gap-4 @[800px]:grid-cols-2">
+        <IntegrationCard
+          description={
+            <>
+              Connect your GitHub account to get rewarded when you contribute to
+              the{' '}
+              <a
+                className="link"
+                href="https://github.com/colorstackorg/oyster"
+                target="_blank"
+              >
+                Oyster codebase
+              </a>
+              .
+            </>
+          }
+          externalUri="https://github.com"
+          isConnected={isGithubConnected}
+          logo="/images/github.svg"
+          name="GitHub"
+          oauthUri={githubOauthUri}
+        />
+      </ul>
+    </ProfileSection>
+  );
+}
+
+type IntegrationCardProps = {
+  description: string | React.ReactElement;
+  externalUri: `https://${string}`;
+  isConnected: boolean;
+  logo: `/images/${string}.svg`;
+  name: string;
+  oauthUri: string;
+};
+
+// TODO: There's a lot of custom styling here that really should be in the
+// design system. We should refactor at some point (we also need an actual
+// design system lol).
+
+function IntegrationCard({
```

Comment:
I updated the UI for the integrations to be formatted like cards. Here's an example of how it looks!

![Screenshot 2024-04-23 at 9 30 14 PM](https://github.com/colorstackorg/oyster/assets/38056800/b6b7e48b-2a94-41c3-b551-981c5e0aaed0)

![Screenshot 2024-04-23 at 9 30 19 PM](https://github.com/colorstackorg/oyster/assets/38056800/f400c643-39f1-4c4d-adf0-6e05f357d427)